### ISSUE TITLE
the size of datasets of video_encoder

### ISSUE THREAD

**Issue by smileto1** [Orange-OpenSource/Cool-Chic#7]
Hello, when using the CIFAR dataset with 1920*1080 resolution YUV 4:2:0 video data and encoding it with the provided script for training, the nn_bpp remains zero, causing the loss function to stay constant and preventing convergence.

**Comment by smileto1** [comment]
![image](https://github.com/Orange-OpenSource/Cool-Chic/assets/66523226/031356c3-f5a6-4222-a99e-ff28aa438383)
Sir.This picture is the printed result of the predicted frame in the code. This is a screenshot of the training process using YUV420 video with 1920*1080 resolution. The value part of this content is all 1. This causes an MSE calculation error and causes the loss to fail to converge. Sorry to bother you, but I can't reproduce the results in your original article.

**Comment by theoladune** [comment]
Hello,

`nn_bpp` refers to the rate associated to the different neural networks. As explained in the [COOL-CHIC paper](https://arxiv.org/abs/2212.05458), it is **not** optimized by the training process, unlike the rate associated to the latent variable `rate_latent_bpp`, so that should not be an issue.

Please join the sequence you'd like to encode and the command line you're using, I'll try it on my side to see what's wrong.

Théo
